wdl/FilterGenotypes.wdl

```diff
@@ -0,0 +1,325 @@
+version 1.0
+
+import "MainVcfQc.wdl" as qc
+import "Structs.wdl"
+import "RecalibrateGq.wdl" as recalibrate_gq
+import "TasksMakeCohortVcf.wdl" as tasks_cohort
+
+workflow FilterGenotypes {
+
+  input {
+    File vcf  # Cleaned GATK-formatted vcf
+    String? output_prefix
+    File ploidy_table
+
+    File gq_recalibrator_model_file
+    Float no_call_rate_cutoff = 0.05
+    Float fmax_beta = 0.5
+
+    # One of the following must be provided
+    File? truth_json  # If given, SL cutoffs will be automatically optimized. Overrides sl_filter_args.
+    String? sl_filter_args  # Explicitly set SL cutoffs. See apply_sl_filter.py for arguments.
+
+    Int optimize_vcf_records_per_shard = 50000
+    Int filter_vcf_records_per_shard = 20000
+
+    # For MainVcfQc
+    Boolean run_qc = true
+    String qc_bcftools_preprocessing_options = "-e 'FILTER~\"UNRESOLVED\" || FILTER~\"HIGH_NCR\"'"
```

Comment:
Would it make more sense to include PASS & MULTIALLELIC in case other filters have been applied?

Reply:
Yes and in that case the user could modify this. Right now we still have the `PESR_GT_OVERDISPERSION`, `BOTHSIDE_SUPPORT`, etc. filters and those were the reason for constructing it this way (fixing that soon though!)